neuromation/cli/formatters/jobs.py

```diff
@@ -211,20 +212,23 @@ def from_job(cls, job: JobDescription) -> "TabularJobRow":
             status=job.status,
             when=when_humanized,
             image=str(job.container.image),
+            owner=("<you>" if job.owner == username else job.owner),
```

Comment:
@mariyadavydova please confirm that replacement self username with `"<you>"` is what we expect. Personally, I prefer to see `"<you>"` rather than my long name `"artemyushkovskiy"` because it's the case of 99% of all my jobs, thus the column full of my name looks frightening.

Reply:
this is better, I think: 
```
$ n ps -a
ID                                        NAME                  STATUS     WHEN    IMAGE            OWNER  DESCRIPTION  COMMAND
job-b65448b2-db2c-4d1a-82f0-edf6a07e262b                        succeeded  May 17  ubuntu:latest    <you>               sleep 10m
job-9ae875f6-19ce-47d4-9606-ebc3b0ac11ca                        succeeded  May 24  ubuntu:latest    <you>               sleep 3
job-47d287ce-90af-46f9-b56f-4eb135893f7c                        succeeded  May 24  ubuntu:latest    <you>               sleep 3
job-11503705-de48-4406-96e6-2a7df52226cb                        succeeded  Jun 27  ubuntu:latest    <you>               
job-f6c8ecc0-aed5-449e-96c5-48613859d645                        succeeded  Jun 27  ubuntu:latest    <you>               
job-29cad754-96c4-429c-ab77-1dc30829414b                        succeeded  Jun 27  ubuntu:latest    <you>               echo 123 ; sleep 10m
job-d98018c9-c4c7-4de5-84ef-b9fe4ae879b0                        failed     Jun 27  python:latest    <you>               python -c 'import time; print(\'.\', end=\'\'); time.sleep(1)
job-1c62ffc6-a7cf-408e-b7b4-26e9aca0f4f2  test-log              succeeded  Jun 27  python:latest    <you>               python -c 'import time; print(100500); time.sleep(10*60)'
job-690ba21a-4a4f-4939-b73c-e1a48f5633a4  test-log              succeeded  Jun 27  python:latest    <you>               python -c 'import time; print(100500); time.sleep(10*60)'
job-e6c96f49-05ce-48d7-b37d-df3f119c2c1b                        succeeded  Jun 27  python:latest    <you>               python -c 'import time; print(100500); time.sleep(10*60)'
job-c85cb6b3-1ab1-421f-8992-15bf2b86313b                        succeeded  Jun 27  python:latest    <you>               python -c 'import time; print(100500); time.sleep(15)'
job-c1545bd2-1fdb-46e0-924b-a75a18c3dc32                        succeeded  Jun 27  python:latest    <you>               python -c 'import time; print(100500); time.sleep(15)'
job-2cd08e17-5825-475e-bad9-b4c80dea068a                        succeeded  Jun 27  python:latest    <you>               python -c 'import time; print(100500); time.sleep(15*60)'
job-ea169c96-fce0-4567-91d0-8c4d32f4ae95  test-log              succeeded  Jun 27  python:latest    <you>               python -c 'import time; print(1); time.sleep(5); print(2)'
job-379cedb0-06b0-4a27-b26f-87443fdf817d  test-log              succeeded  Jun 27  python:latest    <you>               python -c 'import time; print(1); time.sleep(5); print(2)'
job-5d08ee66-0f0a-4f72-850d-7ac84acde2df  test-log              succeeded  Jun 27  python:latest    <you>               python -c 'import time; print(1); time.sleep(5); print(2)'
job-06894d04-172a-4b9b-84a5-ca866e7c268f  test-log              succeeded  Jun 27  python:latest    <you>               python -c 'import time; print(1); time.sleep(1); print(2)'
job-23009f63-e292-4d4c-9293-d54796f63cb4  test-log              succeeded  Jun 27  python:latest    <you>               python -c 'import time; print(1); time.sleep(1); print(2)'
job-e1458f19-2625-4fc4-a1b9-6c3fbd22db02                        failed     Jun 27  python:latest    <you>               python -c 'import time; print(100500); time.sleep(15*60)'
job-4027f150-a7fd-4145-81a4-ca55a6e3f797                        succeeded  Jun 27  python:latest    <you>               python -c 'print(1)'
job-30270d63-b066-46fa-bcb1-05e9573030a1                        succeeded  Jun 27  python:latest    <you>               /bin/bash -c 'echo 1 && sleep 5 && echo 2'
job-6ef7c3c2-f6ef-4e67-acc8-958cd55da356                        succeeded  Jun 27  python:latest    <you>               /bin/bash -c 'echo 1 && sleep 15 && echo 2'
job-a6c97142-aa7c-43b8-b29b-fcd9b870b74e                        succeeded  Jun 27  python:latest    <you>               /bin/bash -c 'echo 1 && sleep 15 && echo 2'
job-ba4506ad-9d93-45f0-945f-414c5375c3b0                        succeeded  Jun 27  python:latest    <you>               python -c 'print(1); import time; time.sleep(5); print(2)'
job-26da5214-0fe0-4b2e-876a-972aaa1c01bb                        succeeded  Jun 27  python:latest    <you>               /bin/bash -c 'echo 1 && sleep 20 && echo 2'
job-0c856eeb-355e-443e-9c7d-4f63df80ccef  test-log              succeeded  Jun 28  python:latest    <you>               bash -c 'echo 1 && sleep 20m && echo 2'
...
```

Reply:
I confirm, `"<you>"` is better here 👍